test/unit/model/test_split_energy.py

```diff
@@ -3,6 +3,9 @@
 # SPDX-License-Identifier: MPL-2.0
 
 import unittest
+from numpy import split
+
+from numpy.core.fromnumeric import mean
```

Comment:
split and mean seem not in use. Can you remove these import  lines?

Reply:
Done